sickbeard/providers/newpct.py

```diff
@@ -110,18 +115,24 @@ def _doSearch(self, search_strings, search_mode='eponly', epcount=0, age=0, epOb
                             try:
                                 if iteration < num_results:
                                     torrent_size = row.findAll('td')[2]
-                                    torrent_row = row.findAll('a')[1]
+                                    torrent_row = row.findAll('a')[0]
 
-                                    download_url = torrent_row.get('href')
-                                    title_raw = torrent_row.get('title')
+                                    showSheetURL = torrent_row.get('href')
                                     size = self._convertSize(torrent_size.text)
-
+                                    title_raw = torrent_row.get('title')
                                     title = self._processTitle(title_raw)
-
-                                    item = title, download_url, size
-                                    logger.log(u"Found result: %s " % title, logger.DEBUG)
-
-                                    items[mode].append(item)
+                                    
+                                    #Search results don't return torrent files directly, it returns show sheets so we must parse showSheet to access torrent.
+                                    data2 = self.getURL(showSheetURL)
```

Comment:
a second getURL for every result is terrible, and will be hella slow parsing results.